dot/sync/chain_sync.go

```diff
@@ -262,6 +262,9 @@ func (cs *chainSync) bootstrapSync() {
 			cs.workerPool.useConnectedPeers()
 			err = cs.requestMaxBlocksFrom(currentBlock, networkInitialSync)
 			if err != nil {
+				if errors.Is(err, errBlockStatePaused) {
+					return
```

Comment:
We can maybe add a log here to know that we are trying to sync while the service is paused? 🤔